docs/src/app/pages/LinkItintro.md

```diff
@@ -0,0 +1,33 @@
+# Platform: LinkIt series boards
+
+The Arduino 101 is a version of the popular Arduino platform based on the Intel&reg; Curie&trade; chipset. It has the same form factor as many other Arduino but supports built-in Bluetooth&reg; low energy.
+
+<div style="text-align: center; font-size: 75%; margin: 16pt 0;">
+![Image of the LinkIt 7697. Source: arduino.cc](/assets/howtos/LinkIt7697.png)
+<br>
+Image source: <a href="https://docs.labs.mediatek.com/resource/linkit7697-arduino/en" target="_blank">LinkIt 7697 on Mediatek Labs</a>
+</div>
+
+<div style="text-align: center; font-size: 75%; margin: 16pt 0;">
+![Image of the LinkIt 7697. Source: arduino.cc](/assets/howtos/LinkIt7697.png)
+<br>
+Image source: <a href="https://www.arduino.cc/en/Main/ArduinoBoard101" target="_blank">Arduino 101 page on Arduino.cc</a>
+</div>
+
+MIT App Inventor provides a sketch for the Arduino 101 and extensions to interact with the sketch. Please follow [these instructions](/assets/tutorials/MIT_App_Inventor_IoT_Setup.pdf) to program your Arduino 101 with the [App Inventor sketch](/assets/resources/AIM-for-Things-Arduino101.zip).
+
+The following extensions are available for use with the companion sketch as part of the [Arduino 101 Extension Bundle](/assets/resources/edu.mit.appinventor.iot.arduino101.aix):
+
+* [Arduino 101 Accelerometer](#/arduino101/arduinoaccelerometer) &mdash; Access the onboard digital 3-axis accelerometer on the Curie's inertial measurement unit (IMU) chip.
```

Comment:
Do all of the Arduino 101 extensions work with the LinkIt? Otherwise you should update this list.